uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-calendar-day.html

```diff
@@ -45,32 +45,36 @@
     <div
       ng-mouseenter="day.toggleTooltip('tooltip', true)"
       ng-mouseleave="day.toggleTooltip('tooltip', false)">
-      <div class="chr_leave-calendar__day-tooltip_item"
+      <div class="chr_leave-calendar__day-tooltip__item"
         ng-repeat="leaveRequest in day.contactData.leaveRequests"
         ng-click="day.tooltip.show = false;
           day.openLeavePopup($event, leaveRequest, null, null, $root.section === 'my-leave')">
-        <strong>{{day.contactData.leaveRequestsAttributes[leaveRequest.id].absenceTypeTitle}}</strong>
+        <div class="chr_leave-calendar__day-tooltip__item_absence-type">
```

Comment:
These changes are not breaking backstop right? Please check.

Reply:
Nope, all seems good. Why would you think it would break BS?